collaboration/wg-showcase-application/README.md

```diff
@@ -0,0 +1,56 @@
+#  Kyma Showcase Application Working Group
+
+Build a Unified Demo Project with a consistent story to showcase a the complete functionality and features of Kyma. This application should be complex enough to showcase advance features, but can also be dumbed down, to showcase only basic, and simple functionalities that can be used in a Getting Started Guide for example.
+
+***Goals***
+- Build a complex enough application to showcase the full functionality of Kyma ( see proposal #506 and [GoogleCloudPlatform example](https://github.com/GoogleCloudPlatform/microservices-demo) as reference) 
+- Split the Application in functional Components ( microservices, micro frontends, serverless functions ) that can be reused and shared across tutorials.
+- Discover and surface missing features or improvements that are needed to offer the best Development Experience possible with Kyma.
+- Document the development process and the learnings using Blogposts, Youtube videos, Live Coding sessions to expand our social media coverage
+- Use new technologies ( focus on buzzwords ) to have a bigger chance of getting viral and create waves in the social network space. Those new technologies ( AI, Machine Learning) can also reveal some of the future proofing necessary for Kyma to be competitive in a broader technology spectrum space.
+- Align existing example repositories to the Unified Story
+    - https://github.com/kyma-incubator/examples
+    - https://github.com/kyma-project/examples
+    - https://github.com/SAP-samples/kyma-runtime-extension-samples
+    - getting started guide
+
+## Contact
+
+* [Slack](https://kyma-community.slack.com/archives/C01C40T8CKD)
+* [Open community issues and pull requests](https://github.com/kyma-project/community/labels/area%2Fcommunity)
+* [Project Board](https://github.com/kyma-project/community/projects/5)
+
+## Meetings
+
+* [Thursday at 16:00 CEST](https://calendar.google.com/calendar/ical/3abtp9lh0mn3iiov5772jftccc%40group.calendar.google.com/public/basic.ics)
```

Comment:
Remember to update this on Sunday to make it `CET` :) 

Also, consider adding a time converter: [for CEST](https://www.thetimezoneconverter.com/?t=16:00&tz=CEST%20(Central%20European%20Summer%20Time)), [for CET](https://www.thetimezoneconverter.com/?t=16:00&tz=CET%20(Central%20European%20Time)).